docs/bnb_smart_chain/staking/index.md

```diff
@@ -0,0 +1,191 @@
+- **Delegate Voting Power**: The flag to indicate whether the delegator would like to delegate his/her voting power
+  to the validator for governance.
+
+The `Delegate` transaction will deduct the amount of BNB from the delegator address and issue the corresponding staking
+credit to the validator. The delegator will then share the rewards and the risks with the validator, according to the
+commission rate and the slashing rate of the validator.
+
+The credit tokens (or share) a delegator will get is calculated as: `delegation amount` * `total supply of credit token` / `total pooled BNB`.
+The `total pooled BNB` includes the delegation BNB and unclaimed reaward BNB of the validator. 
+It means that a delegator will get credit tokens based on the ratio of his/her delegation BNB amount to the total staked and reward BNB.
+
+When the validator receives the block reward, the "total pooled BNB" amount will increase. This increase benefits the
+delegator when they unbond, as they will receive their delegation along with BNB rewards from the pool.
+
+### Redelegate
+
+To redelegate BNB from one validator to another, a delegator needs to send a `Redelegate` transaction to the `StakeHub`
+contract, specifying the following information:
+
+- **Source operator address**: The address of the source validator, which will send the BNB to the destination
+  validator.
+- **Destination operator address**: The address of the destination validator, which will receive the BNB from the
+  source validator.
+- **Amount**: The amount of BNB that the delegator wants to redelegate from the source validator to the destination
+  validator.
+- **Delegate Voting Power**: The flag to indicate whether the delegator would like to delegate his/her voting power
+  to the destination validator for governance.
+
+The `Redelegate` transaction will deduct the amount of staking credit from the source validator and issue the
+corresponding staking credit to the destination validator. The delegator will then share the rewards and the risks with
+the destination validator, according to the commission rate and the slashing rate of the destination validator.
+
+The `Redelegate` transaction does not incur the unbonding period, but it will incur the redelegation fee,
+which is designed to prevent delegators from frequently switching between validators to chase
+the highest rewards or avoid the highest risks. The current fee rate is 0.002%.
+
+### Undelegate
+
+To undelegate BNB from a validator, a delegator needs to send an `Undelegate` transaction to the `StakeHub` contract,
+specifying the following information:
+
+- **Operator address**: The address of the validator, which will send the BNB to the delegator.
+- **Amount**: The amount of BNB that the delegator wants to unstake from the validator.
+
+The `Undelegate` transaction will deduct the amount of staking credit from the validator and return the corresponding
```

Comment:
==> `The `Undelegate` transaction will burn the amount of staking credit from the user`

Reply:
fixed